src/blake2b/blake2b64.ts

```diff
@@ -201,22 +274,28 @@ export function blake2b(input: Uint8Array, key: Uint8Array | null = null, outlen
  * Assumes no key
  */
 // @ts-ignore
-@inline()
+@inline
 export function blake2bResetForShortMessage(ctx: Context): void {
+  const h = ctx.h;
   // Initialize State vector h with IV
-  for (let i = 0; i < 8; i++) {
-    unchecked(ctx.h[i] = BLAKE2B_IV[i]);
-  }
+  // unchecked(h[0] = BLAKE2B_IV_0);
+  unchecked(h[1] = BLAKE2B_IV_1);
+  unchecked(h[2] = BLAKE2B_IV_2);
+  unchecked(h[3] = BLAKE2B_IV_3);
+  unchecked(h[4] = BLAKE2B_IV_4);
+  unchecked(h[5] = BLAKE2B_IV_5);
+  unchecked(h[6] = BLAKE2B_IV_6);
+  unchecked(h[7] = BLAKE2B_IV_7);
 
   // Danger: These operations and resetting are really only possible because our input is exactly 128 bytes
   // for(let i = 0; i < 128; i++) {
   //   ctx.b[i] = input[i];
   // }
-  // ctx.b = input; 
+  // ctx.b = input;
   // We don't actually have to reset these as they are overwritten and we never stream..
   // ctx.b.set(input);
   // ctx.m.fill(0);
   // ctx.v.fill(0);
 
-  ctx.h[0] ^= 0x01010000 ^ (ctx.outlen as u64);
+  h[0] = BLAKE2B_IV_0 ^ 0x01010000 ^ (ctx.outlen as u64);
```

Comment:
Nit: You could move this up to where  ```// unchecked(h[0] = BLAKE2B_IV_0);``` currently is (line 281), and make it unchecked